app/controllers/adjustments_controller.rb

```diff
@@ -0,0 +1,37 @@
+class AdjustmentsController < ApplicationController
+  include Administrated
+  layout 'admin'
+  before_action :set_adjustment, only: [:update]
+  
+  def edit
+    @adjustment = Adjustment.find(params[:id])
+  end
+
+  # PATCH/PUT /adjustments/1
+  # PATCH/PUT /adjustments/1.json
+  def update
+    @adjustment = Adjustment.find(params[:id])
+    respond_to do |format|
+      puts "----------------reason--#{@adjustment.reason}"
```

Comment:
you are right. that wasn't meant to be pushed actually. was a quick debug. will remove it.